apt/private/deb_resolve.bzl

```diff
@@ -120,7 +124,7 @@ def _deb_resolve_impl(rctx):
             # NOTE: the split("~") is needed when we run bazel from another
             # directory, e.g. when running e2e tests we change dir to e2e/smoke
             # and then rctx.name is 'rules_distroless~~apt~bullseye'
-            repo_name = rctx.name.split("~")[-1].replace("_resolve", ""),
```

Comment:
we have this utility here https://github.com/GoogleContainerTools/rules_distroless/blob/81c2dfda1e36d6ecc16ea1d7d76e624ddd2f8f09/apt/private/util.bzl#L26C5-L26C19. can we use that instead?

Reply:
ping